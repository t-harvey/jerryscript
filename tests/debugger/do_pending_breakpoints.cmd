fbreak f
list pending
n
n
c
c
