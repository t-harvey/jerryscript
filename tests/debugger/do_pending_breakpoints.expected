Connecting to: localhost:5001
Stopped at tests/debugger/do_pending_breakpoints.js:15
(jerry-debugger) fbreak f
Breakpoint not found
Pending breakpoint added
(jerry-debugger) list pending
0: f
(jerry-debugger) n
Stopped at tests/debugger/do_pending_breakpoints.js:17
(jerry-debugger) n
Breakpoint 1 at <unknown>:1 (in f() at line:1, col:1)
Stopped at tests/debugger/do_pending_breakpoints.js:19
(jerry-debugger) c
Stopped at breakpoint:1 <unknown>:1 (in f() at line:1, col:1)
(jerry-debugger) c
Connection closed.
